### PR TITLE
Automatically port Reflection.PortableExecutable.CodeViewDebugDirectoryData

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

File: xml/System.Reflection.PortableExecutable/CodeViewDebugDirectoryData.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Iteration of the PDB. The first iteration is 1. The iteration is incremented each time the PDB content is augmented.</summary>
```
Comment (rpetrusha) [inline xml/System.Reflection.PortableExecutable/CodeViewDebugDirectoryData.xml]:
```suggestion
        <summary>The iteration of the PDB. The first iteration is 1. The iteration is incremented each time the PDB content is augmented.</summary>
```

File: xml/System.Reflection.PortableExecutable/CodeViewDebugDirectoryData.xml [hunk 2]
```diff
@@ -72,7 +72,7 @@
         <ReturnType>System.Guid</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>GUID (Globally Unique Identifier) of the associated PDB.</summary>
```
Comment (rpetrusha) [inline xml/System.Reflection.PortableExecutable/CodeViewDebugDirectoryData.xml]:
```suggestion
        <summary>The Globally Unique Identifier (GUID) of the associated PDB.</summary>
```

File: xml/System.Reflection.PortableExecutable/CodeViewDebugDirectoryData.xml [hunk 3]
```diff
@@ -97,7 +97,7 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Path to the .pdb file containing debug information for the PE/COFF file.</summary>
```
Comment (rpetrusha) [inline xml/System.Reflection.PortableExecutable/CodeViewDebugDirectoryData.xml]:
```suggestion
        <summary>The path to the .pdb file that contains debug information for the PE/COFF file.</summary>
```

**Comment by carlossanlop** [comment]
@rpetrusha thanks for the type summary and remark. I added them.

@GrabYourPitchforks @steveharter can you please review this last commit? The type is not documented in code so I would like to make sure we're describing it accurately.

**Comment by rpetrusha** [comment]
Is this ready to merge, @carlossanlop?

**Comment by carlossanlop** [comment]
Yes, ready to merge @rpetrusha.

**Pull request merged**